### PR TITLE
Add basic page structure

### PR DESCRIPTION
**Pull request by pasytchangwa**
- Install Webpack 
- Added basic page structure
- Added setup files

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,41 @@
...
+     <input type="text" placeholder="Your name" aria-label="name" class="name-input">
+    </div>
+    <div class="score">
+     <input type="text" placeholder="Your score" aria-label="name" class="score-input">
+    </div>
+    <button type="submit" class="submit-btn">Submit</button>
+   </form>
+  </main>
+  <footer class="footer-section"></footer>
+ </div>
```
Comment (vichuge) [inline src/index.html]: - [ ] index.html main contain only root element. In other words, the code you made is excellent, now let's convert it to js code inside index.js. 🐶 
- hint: Try to use `<div id="root"></div>` on index.html and use the index.js to create the project with js code.
- Don't forget to use ES6 best practices.

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] A good readme file could help you to have a professional presentation on your project, I kindly invite you to add one following the [approved template](https://github.com/microverseinc/readme-template). 🐱 
image

- [ ] Check the comments under the review.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -1 +1,112 @@
-import _ from 'lodash';
+// import _ from 'lodash';
```
Comment (bellom) [inline src/index.js]: Kindly remove commented code before merging.


It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by bellom — APPROVED** [review]
## Hi @pasytchangwa :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best**

**Pull request merged**